test/integration/node-specific/client_close.test.ts

```diff
@@ -431,13 +433,17 @@ describe.skip('MongoClient.close() Integration', () => {
         const metadata: MongoDBMetadataUI = { requires: { topology: 'sharded' } };
 
         describe('after SRVPoller is created', () => {
-          it('timers are cleaned up by client.close()', metadata, async () => {
+          it.skip('timers are cleaned up by client.close()', metadata, async () => {
             const run = async function ({ MongoClient, expect, getTimerCount }) {
               const SRV_CONNECTION_STRING = `mongodb+srv://test1.test.build.10gen.cc`;
+
               // 27018 localhost.test.build.10gen.cc.
               // 27017 localhost.test.build.10gen.cc.
 
-              const client = new MongoClient(SRV_CONNECTION_STRING);
+              const client = new MongoClient(SRV_CONNECTION_STRING, {
+                serverSelectionTimeoutMS: 2000,
```

Comment:
Can you add a comment explaining why this test specifically needs a low serverSelectionTimeout and tls: false?